lib/internal/Magento/Framework/Filesystem/Io/File.php

```diff
@@ -467,7 +470,7 @@ public function read($filename, $dest = null)
      */
     public function write($filename, $src, $mode = null)
     {
-        if (is_string($src) && @is_readable($src)) {
+        if (is_string($src) && ctype_print($src) && @is_readable($src)) {
```

Comment:
Can we revert this? The goal on this one is literally just to mark classes/interfaces as API and not make any code change. If this is required because of `strict_types=1`, let's remove it and create a separate PR for it.